### PR TITLE
#184637233 Set up an empty Express Boilerplate with dotenv

### PR DESCRIPTION
**Pull request by masterchief-00**
#### What does this PR do?

- Creates a boilerplate for the project
- added dotenv configuration

#### Description of Task to be completed?
 Set up an empty Express Boilerplate with dotenv
#### How should this be manually tested?
 clone the project and use the terminal to start the development server by "npm run dev"
#### Any background context you want to provide?
 none
#### What are the relevant pivotal tracker/Trello stories?
 Tracker: 184637233
#### Screenshots (if appropriate)
 none
#### Questions:
 none

### CODE REVIEW

**Review by kirengaboris — APPROVED** [review]

File: .env.example [hunk 1]
```diff
@@ -0,0 +1 @@
+PORT=<port number>
```
Comment (fab-ryan) [inline .env.example]: you need to add new line after

File: src/helpers/helperFn.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+/** eg:HELPER FUNCTION 1 
+ * not a real file
+*/
```
Comment (fab-ryan) [inline src/helpers/helperFn.js]: adding new line here

File: src/validations/validation.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+/** eg:VALIDATIONS 
+ * not a real file
+*/
```
Comment (fab-ryan) [inline src/validations/validation.js]: leave new here

**Review by fab-ryan — CHANGES_REQUESTED** [review]
check those comments

File: .env.example (hunk 1 above)
Comment (masterchief-00) [reply .env.example]: done

File: src/helpers/helperFn.js (hunk 2 above)
Comment (masterchief-00) [reply src/helpers/helperFn.js]: done

File: src/validations/validation.js (hunk 3 above)
Comment (masterchief-00) [reply src/validations/validation.js]: done

**Review by fab-ryan — APPROVED** [review]

**Review by istafrica-dennw — CHANGES_REQUESTED** [review]
commit message do not make much sense becae they are verbose. too much long. I ma expecting shorter description

chore (): setup an empty express boilerplate
You can leave the () empty as it is not active over a single file/folder

**Comment by masterchief-00** [comment]
> commit message do not make much sense becae they are verbose. too much long. I ma expecting shorter description
> 
> chore (): setup an empty express boilerplate You can leave the () empty as it is not active over a single file/folder

I have edited the commit message

**Comment by fab-ryan** [comment]
then squash commits

**Comment by masterchief-00** [comment]
> then squash commits

I think they are squashed because I did the "git rebase -i HEAD~3" and now **git log** shows one commit

**Comment by fab-ryan** [comment]
that means you only have more than 3 header
check again

**Review by fab-ryan — APPROVED** [review]

**Review by istafrica-dennw — CHANGES_REQUESTED** [review]
We need only one Commit, or maximum 2

**Review by istafrica-dennw — APPROVED** [review]

**Pull request merged**